### DISCUSSION TITLE
How to configure production setup ?

### DISCUSSION THREAD

**Discussion by RocketRene** [supabase/supabase#5228] [Questions]
Hi, Can Someone help me with the docker installation, or better walk me through it. I don't get it.
I have changed the secrets in the .env files but I am not sure what I should do with the urls
I want to access it on supabase.renekuhn.com and I can but nothing is working. I cant access the db or the API section
What do I have write here ?
![image](https://user-images.githubusercontent.com/55456939/151670758-342c39e0-06d5-4e6c-aedf-f2c635fae843.png)

**Comment by heromiyo** [comment]
you need to setup nginx as a reverse proxy

**Reply by RocketRene** [reply]
Thank you
